src/discordBot.py

```diff
@@ -68,7 +68,10 @@ async def addalias(ctx, *args):
 
             leaderboardBot.addAlias(alias, name)
             leaderboardBot.updateAlias()
-            await ctx.send(f'{alias} is now an alias for {name}')
+            if alias in leaderboardbot.alias.keys() and leaderboardbot.alias[alias] == name:
```

Comment:
because adding the alias calls `upadateAlias` this is my check that the item was added to the database successfully